graph/appender/dead_node.go

```diff
@@ -3,14 +3,20 @@ package appender
 import (
 	"github.com/kiali/kiali/graph"
 	"github.com/kiali/kiali/services/business"
+	"github.com/kiali/kiali/services/models"
 )
 
 // DeadNodeAppender is responsible for removing from the graph unwanted nodes:
 // - nodes for which there is no traffic reported and the related schema is missing
 //   (presumably removed from K8S). (kiali-621)
 // - service nodes for which there is no incoming error traffic and no outgoing
-//   edges (kiali-1326)
-type DeadNodeAppender struct{}
+//   edges (kiali-1326) - excluding in this rule the nodes representing egress
```

Comment:
is this related to kiali-1326? I see "- excluding in this rule..." appears right after that. Do you mean to put "- excluding in this rule..." on a new line? Otherwise, its confusing - looks like this is related to kiali-1326 (unless this is related to 1326, in which case, ignore me :)

Reply:
I think is related. In kiali-1326 _terminal service nodes_ that has NO errors are marked as dead nodes and excluded from the graph.

But egress traffic is recognized as service nodes in our code and they are terminal nodes. They matches with the logic of kiali-1326. So, in kiali-1526 I'm preventing these "egress nodes" from being identified as dead nodes.